adhocracy4/categories/dashboard.py

```diff
@@ -14,8 +14,23 @@ class ModuleCategoriesComponent(ModuleFormSetComponent):
 
     form_title = _("Edit categories")
     form_class = forms.CategoryFormSet
+
     form_template_name = "a4categories/includes/module_categories_form.html"
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["category_alias_form"] = forms.CategoryAliasForm()
+        return context
+
+    def get(self, request, *args, **kwargs):
+        self.object = None
+        form_class = self.get_form_class()
+        form = self.get_form(form_class)
+        category_alias_form = forms.CategoryAliasForm()
+        return self.render_to_response(
+            self.get_context_data(form=form, category_alias_form=category_alias_form)
+        )
+
```

Comment:
@fuzzylogic2000 @Rineee this is my attempt to add the second form to the context, tested in mB https://github.com/liqd/a4-meinberlin/pull/5095/files however currently failing, does either one of you want to tag in?